components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -336,11 +368,14 @@ class PromptFeature private constructor(
             when (promptRequest) {
                 is File -> filePicker.handleFileRequest(promptRequest)
                 is Share -> handleShareRequest(promptRequest, session)
+                is SelectCreditCard -> {
+                    if (promptRequest.creditCards.isNotEmpty()) {
+                        creditCardPicker?.handleSelectCreditCardRequest(promptRequest)
+                    }
+                }
                 is SelectLoginPrompt -> {
                     if (promptRequest.logins.isNotEmpty()) {
-                        loginPicker?.handleSelectLoginRequest(
-                            promptRequest
-                        )
+                        loginPicker?.handleSelectLoginRequest(promptRequest)
```

Comment:
Since you're changing the naming for other login references, should we change this to `selectPicker` or something of the sort?